demophin.py

```diff
@@ -155,11 +182,17 @@ def error404(error):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) > 1:
+    if len(sys.argv) > 1 and not sys.argv[1].startswith('-'):
```

Comment:
If you're doing something more complicated than a single argument, just use argparse.